main.go

```diff
@@ -133,3 +147,5 @@ func getUnrestrictedClient() client.Client {
 	}
 	return k8sClient
 }
+
+func pluginAddToScheme(scheme *runtime.Scheme) {}
```

Comment:
Add a comment saying that this function is a placeholder for offering plugin integration